test/src/main/scala/scalarules/test/compiler_plugin/KindProjected.scala

```diff
@@ -3,4 +3,4 @@ package scalarules.test.compiler_plugin
 import scala.language.higherKinds
 
 class HKT[F[_]]
-class KKTImpl extends HKT[Either[String, ?]]
\ No newline at end of file
+class KKTImpl extends HKT[Either[String, *]]
```

Comment:
This [commit](https://github.com/scala/scala/pull/10005/files) has broken compilation for this file since Scala `2.12.16` where `?` now has a different purpose in Scala 3. Found out that reverting to `*` was the simplest workaround to avoid bringing `-Xsource:3` compiler flags just yet.

See https://docs.scala-lang.org/scala3/guides/migration/plugin-kind-projector.html